config/initializers/flipper.rb

```diff
@@ -59,4 +77,8 @@
 end
 
 # Add known feature toggles. This may fail if the database isn't setup yet.
-OpenFoodNetwork::FeatureToggle.setup! rescue ActiveRecord::StatementInvalid
+begin
+  OpenFoodNetwork::FeatureToggle.setup!
+rescue StandardError
+  ActiveRecord::StatementInvalid
+end
```

Comment:
I'm not familiar with the short-hand rescue format, and now that it's been expanded I see that it doesn't do what may have been intended. 
`±` It looks like it was meant to catch only the `ActiveRecord::StatementInvalid` error, and return nothing.

Reply:
Good pick, it was some Rubocop autoformating, not sure how I missed this ?